.github/container/Dockerfile.base

```diff
@@ -93,12 +100,6 @@ ADD install-efa.sh /usr/local/bin
 ENV LD_LIBRARY_PATH=/opt/amazon/efa/lib:${LD_LIBRARY_PATH}
 ENV PATH=/opt/amazon/efa/bin:${PATH}
 
-###############################################################################
-## Emergency fix: nsys not in PATH
-###############################################################################
-
-RUN ln -s /opt/nvidia/nsight-compute/*/host/target-linux-x64/nsys /usr/local/cuda/bin
```

Comment:
Is this version of ncu still in the base image?

Reply:
The base image contains both ncu and nsys, at versions older than the newest available in the devtools repository.